src/main/java/br/com/moip/resource/Amount.java

```diff
@@ -3,11 +3,31 @@
 public class Amount {
 
     private Integer total;
+    private Integer fees;
+    private Integer refunds;
+    private Integer liquid;
+    private String currency;
 
     public Integer getTotal() {
         return total;
     }
 
+    public Integer getFees() {
+        return fees;
+    }
+
+    public Integer getRefunds() {
+        return refunds;
+    }
+
+    public Integer getLiquid() {
+        return liquid;
+    }
+
+    public String getCurrency() {
+        return currency;
+    }
+
     @Override
     public String toString() {
         return new StringBuilder("Amount{")
```

Comment:
@vkdrn Thanks for contributing!! Would you mind to add all attributes in `toString`method too?